src/components/elements/accordion.tsx

```diff
@@ -93,15 +118,35 @@ const ItemExpanded = ({ is_default_open, child, child_idx, nodes, id }) => {
         else setHeight(0)
     })
 
-    React.useEffect(() => {
+    useEffect(() => {
         if (is_default_open) setExpanded(true)
     }, [])
 
-    React.useEffect(() => child && setHeight(getHeight(child_idx)), [is_expanded])
+    useEffect(() => child && setHeight(getHeight(child_idx)), [is_expanded])
+
+    const deployer = is_expanded ? (
+        <img src={Minus} alt="Minus" height="16" width="16" />
+    ) : (
+        <img src={Plus} alt="Plus" height="16" width="16" />
+    )
```

Comment:
```suggestion
    const deployer = <img src={is_expanded ? Minus : Plus} alt="Minus" height="16" width="16" />
```
Can it be done this way?

Reply:
Done by this way, thank you 🙇